charger/ocpp/connector.go

```diff
@@ -122,6 +113,21 @@ func (conn *Connector) TransactionID() (int, error) {
 	return conn.txnId, nil
 }
 
+func (conn *Connector) StatusOCPP() (core.ChargePointStatus, error) {
+	conn.mu.Lock()
```

Comment:
Reihenfolge Lock/Connected?

Reply:
Das ist in zahlreichen Funktionen bereits unterschiedlich (nicht von mir).

Wie soll es sein? Änder es am besten gerade so ab wie du es am schönsten findest.
Macht das Diff größer. Oder separater PR?

Reply:
Zumindest die neuen Funktionen dürfen ja in sich konsistent sein. Gerne nach Mehrheit der Ist-Implementierung.